crates/forge/src/scarb.rs

```diff
@@ -43,6 +43,14 @@ pub struct ForgeConfig {
     #[serde(default)]
     /// Should runner exit after first failed test
     pub exit_first: bool,
+    pub fork: Option<Vec<PredefinedFork>>,
+}
+
+#[derive(Deserialize, Debug, PartialEq, Default, Clone)]
+pub struct PredefinedFork {
+    pub name: String,
+    pub url: String,
+    pub block_id: HashMap<String, String>,
```

Comment:
I think we should validate `block_id` after parsing, that is:
1. It has only one key pair inside
2. The key belongs to set {"number", "hash", "tag"}

Would also add unit tests for that